config/initializers/cors.rb

```diff
@@ -5,12 +5,21 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+ALLOWED_CORS_ORIGINS = [
```

Comment:
should prob. create controller tests for this (e.g., one test that allows openstax.org, and one that doesnt (a mock that raises a cors exception)

Reply:
I think these got created Jan 5th, resolving,